src/pages/homepage/homepage.css

```diff
@@ -0,0 +1,33 @@
+.homepage-container{
+    padding: 100px;
```

Comment:
use rem unit for achieve responsiveness

Reply:
Done [21c1c43](https://github.com/shivanigangadharan/streambucket/pull/4/commits/21c1c43d5b49d9b08c2469a473d8cb79db95cb87)